share/cmake/macros/TargetUtils.cmake

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+#
+# Imported from the OpenImageIO project.
+#
+
+###########################################################################
+# Macro to install targets to the appropriate locations.  Use this instead
+# of the install(TARGETS ...) signature. Note that it adds it to the
+# export targets list for when we generate config files.
+#
+# Usage:
+#
+#    install_targets (target1 [target2 ...])
+#
+macro (install_targets)
```

Comment:
Reading this again, I wonder if we should drop the COMPONENT here as we don't use any. Also on a related note, the OpenColorIO library target seems to be installed twice, one in the CMakeLists.txt just before the macro, the other here. Doesn't look like it's causing any issues but it maybe could be improved.

We could also drop the macro as it could be confusing, we only use that in 2 places for the public targets, all others targets are installed the usual way.

I can push some changes to address that.

Any thoughts @lgritz ?

Reply:
This is just a convenience macro. If it's not exactly what you need or not used in enough places to aid in simplifying your cmake scripts, just skip it and use `install()` directly. I don't remember/understand exactly what the COMPONENT bits do, I think maybe I wasn't even the person who wrote this macro or added that argument.

Reply:
Thanks for the feedback, I implemented the change, hopefully this simplify things.